src/MacroTools/QuestSystem/UtilityStructs/ObjectiveHeroWithLevelReachRect.cs

```diff
@@ -16,7 +16,7 @@ public ObjectiveHeroWithLevelReachRect(int targetLevel, Rectangle targetRect, st
       _targetRect = targetRect.Rect;
       _targetLevel = targetLevel;
       region target = RectToRegion(_targetRect);
-      Description = $"Any hero reaches {rectName}";
+      Description = $"Any hero that is atleast level {targetLevel} reaches {rectName}";
```

Comment:
```suggestion
      Description = $"Any level {targetLevel}+ hero reaches {rectName}";
```
This is a bit shorter and we have very limited space at times.